drape_frontend/gui/layer_render.cpp

```diff
@@ -185,6 +185,10 @@ class ScaleFpsLabelHandle : public MutableLabelHandle
 
     auto const vs = static_cast<float>(df::VisualParams::Instance().GetVisualScale());
     m2::PointF offset(10.0f * vs, 30.0f * vs);
+#ifdef OMIM_OS_IPHONE
+    // Move below dynamic island.
```

Comment:
What is dynamic island?

Reply:
The iPhone camera cutout in the screen is called Dynamic Island by Apple